### ISSUE TITLE
Fixed number when inference in topology result

### ISSUE THREAD

**Issue by anhtr-nguyn** [Franpin/TopoLogic#10]
Hi author,

Thank you for your wonderful work, it's really make sense and meaningful, insightful about the topological reasoning.

I am wondering about the results when we are on testing mode. 

Your source code will hard code the lamda_1, lamda_2, P and W instead of calling the trained value.

`# P = self.transformer.decoder.P.cpu()
        # w = self.transformer.decoder.w.cpu()
        # lamda_1 = float(self.transformer.decoder.lamda_1.cpu())
        # lamda_2 = float(self.transformer.decoder.lamda_2.cpu())

        P = 2
        w = 11.5275
        lamda_1 = 1
        lamda_2 = 1

        topo = torch.exp(-torch.pow(topo,P)/(w))*topo_mask

        distance_topo = topo.detach().cpu().numpy()
        Sim_topo = preds_dicts['all_lclc_preds'][-1].squeeze(-1).sigmoid().detach().cpu().numpy()
        all_lclc_preds = lamda_1*Sim_topo + lamda_2*distance_topo
`

Why did you do it ? Your just the performance when fixed the number is better ? Are they the same ?
